flightmate/src/main/webapp/dashboard.jsp

```diff
@@ -14,5 +14,7 @@
 		<p>Hello ${user.getFirstName()}</p>
 		<p>Your role: ${user.getRole().toString()}</p>
 	</main>
+	 <a href="flight.jsp">Manage Flights</a>
```

Comment:
Once Jianxiang's branch is merged, we'll have to set this to be for Administrators only.